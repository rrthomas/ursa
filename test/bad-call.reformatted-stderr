Error: Line 3, col 5:
  2 | let g = fn(): Int {
> 3 |     h()
          ^~~
  4 | }

Invalid call

Traceback (most recent call last)
  line 6
        g(), in f
  line 8
    f(), at top level