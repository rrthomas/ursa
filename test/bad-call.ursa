let h = 3
let g = fn(): Int { h() }
let f = fn(): Int { g() }
f()
