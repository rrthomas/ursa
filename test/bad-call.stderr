Error: Line 2, col 21:
  1 | let h = 3
> 2 | let g = fn(): Int { h() }
                          ^~~
  3 | let f = fn(): Int { g() }

Invalid call

Traceback (most recent call last)
  line 3
    let f = fn(): Int { g() }, in f
  line 4
    f(), at top level